### DISCUSSION TITLE
Build errors if old configuration files are used.

### DISCUSSION THREAD

**Discussion by deezee24** [mriscoc/Ender3V2S1#803] [General]
I’m try to compile the new firmware for ender3 v2 4.2.2. With t13 ,la,is
But I’m getting an error and it’s saying special configuration is outdated.  Any help would be appreciated.  I am very new to this.  Thanks

**Comment by mriscoc** [comment]
Use the Configurator with the updated configuration files directly from the repository.

![image](https://user-images.githubusercontent.com/2745567/226133978-239ba12a-bbae-414a-b36a-e1464176d0bc.png)

**Comment by deezee24** [comment]
im am getting the zip files from the repository. the special configuration zip file im using has a date of dec.2022. its at the bottom of the list.  here is a screen shot of the errors im getting
<img width="2237" alt="Screenshot 2023-03-18 at 11 45 13 AM" src="https://user-images.githubusercontent.com/125142219/226116105-bc210972-3b55-416d-8df2-7908bec147b3.png">

**Reply by mriscoc** [reply]
Your configuration files are old, the last ones in the repository are from this month.

**Comment by deezee24** [comment]
I’m not sure how you got to that page. I can only find the files on the special configurations page. I highlighted the the zip file I was downloading.  It needs to be a zip file not a bin. Correct?  Sorry for being a pain. 
![5CE0359B-538E-49CA-9752-4611F5843546](https://user-images.githubusercontent.com/125142219/226148834-b6ebb8ac-8a46-4e41-b238-d3683d618dd2.jpeg)

**Reply by mriscoc** [reply]
That zip file is from the original old publication. You need to download from https://github.com/mriscoc/Special_Configurations look for the big green button named "<> Code"

**Reply by I-wish-2** [reply]
Thank you Miguel as I was having the same issue with the current release during compiling.  Thanks for pointing to the answer.
Doug
